scripts/src/java/org/oppia/android/scripts/docs/KDocValidityCheck.kt

```diff
@@ -1,4 +1,4 @@
-package org.oppia.android.scripts.kdoc
+package org.oppia.android.scripts.docs
```

Comment:
File: ``KdocValidityCheck``

(note camel-casing/capitalization). Ditto for the proto message name.

We consider "kdoc" to be one word which means as a unit it's capitalized/lowercase rather than being partly.

Reply:
Done.